### PR TITLE
Project settings updates

### PR DESCRIPTION
**Pull request by bryanluby**
This pr has a few project settings updates:

- Bump travis image to Xcode 10.1
- Remove explicit swift version from podspec

@vokal/ios-developers review please

### CODE REVIEW

File: VOKUtilities.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "VOKUtilities"
-  s.version          = "0.12.0"
+  s.version          = "0.12.1"
```
Comment (designatednerd) [inline VOKUtilities.podspec]: I'd recommend making this `0.13` if you're doing the main compilation in Swift 4.2 mode

Comment (bryanluby) [reply VOKUtilities.podspec]: Yep good call

**Comment by bryanluby** [comment]
Actually going to close this as I'm hitting multiple walls getting `pod lib lint` to work nicely with the latest Xcode/Mojave. This can be revisited later after CocoaPods 1.6 is out.

**Review by vmammanalupo — APPROVED** [review]
This LGTM 👍

**Comment by bryanluby** [comment]
@vokal/ios-developers merging this - needed to set an explicit Swift 5 version to get things to lint. Also updated to 10.2 and updated some gems.

**Pull request merged**